src/foam/core/JSON.js

```diff
@@ -481,15 +481,26 @@ foam.LIB({
     },
 
     function parseString(jsonStr, opt_ctx) {
-      return this.parse(eval('(' + jsonStr + ')'), undefined, opt_ctx);
+        console.debug('parseString: '+jsonStr);
```

Comment:
This shouldn't be necessary, the call in HTTPBox to foam.json.parse() is wrong, and if we fix that, then we shouldn't need this change.